packages/rulesets/src/spectral/test/parameters-in-post.test.ts

```diff
@@ -0,0 +1,158 @@
+import { Spectral } from "@stoplight/spectral-core"
+import linterForRule from "./utils"
+
+let linter: Spectral
+
+beforeAll(async () => {
+  linter = await linterForRule("ParametersInPost")
+  return linter
+})
+
+test("ParametersInPost should find errors", () => {
```

Comment:
>ParametersInPost [](http://example.com/codeflow?start=6&length=16)

some more tests to consider

1. More than  one query params which are not api-version
2. Query params with only api-version should not flag errors